components/StopDialog.tsx

```diff
@@ -0,0 +1,46 @@
+import { useEffect, useState } from 'react';
+import Dialog from 'components/Dialog/Dialog';
+import s from 'stylesheets/StopDialog.module.scss';
+
+const StopDialog = (): React.ReactElement | null => {
+  const [isOpen, setOpen] = useState<boolean>(false);
+
+  useEffect(() => setOpen(true), []);
+
+  const handleDismiss = async () => {
+    setOpen(false);
+  };
+
+  if (isOpen)
+    return (
+      <Dialog
+        title="You shouldn't make edits here any more"
+        isOpen={true}
+        onDismiss={() => setOpen(false)}
+        showCloseButton={false}
+        className={s.stopDialog}
+      >
+        <>
+          <p>
+            This is a child resident, so you must only use Mosaic for them from
+            now on.
```

Comment:
Should we maybe also include in the copy that the record may be out of date and should not be relied upon?